### DISCUSSION TITLE
Sugestoes

### DISCUSSION THREAD

**Discussion by ccommim** [RamonSilva20/mapos#2245] [Ideas]
O sistema e bacana d+, e gostaria de contribuir com algumas sugestoes interessantes:

-Cadastro de clientes poderia tem um espaco para anotacoes;
-Poderia existir um cadastro de Tecnicos;
-Poderia ser possivel adicionar mais de um tecnico na OS;

Abraco,

**Comment by KelvinEger** [comment]
> O sistema e bacana d+, e gostaria de contribuir com algumas sugestões interessantes:
> 
> -Cadastro de clientes pode ter um espaço para anotações; -Poderia existir um cadastro de Técnicos; -Poderia ser possível adicionar mais de um técnico no OS;
> 
> Abraco,

Estou implementando novas funcionalidades!
Além do espaço para anotações, estou implementando também geração de NF-e
